.travis.yml

```diff
@@ -4,13 +4,14 @@ python:
   - 2.6
   - 2.7
   - 3.5
+cache: pip
 matrix:
   allow_failures:
     - python: 3.5
 install:
+  - pip install -U pip coveralls wheel
   - pip install https://github.com/thijstriemstra/python-messaging/archive/master.zip
-  - pip install coveralls
-  - pip install wheel
+  - pip install -r requirements.txt
```

Comment:
Ok, @jerith's pointed me at where the problem is, in py35 the `filter()` function returns some `<filter object at 0x1088f3898>` instead of a list and when that's passed as `install_requires` it's silently ignored. Explicitly doing `list(filter( ... ))` when loading the `requirements.txt` fixes it.

Best way to solve this is probably to move all the requirements to the `setup.py` file and having a single `-e .` entry in the `requirements.txt` file.